docs/examples/custom_operations/custom_operator/naive_histogram/README.md

```diff
@@ -0,0 +1,48 @@
+# DALI custom operator example
+
+This is an example/template of a DALI Custom operator.
+The implemented operation is a (very) naive histogram on a grayscale image.
+The NaiveHistogram operator is a GPU-only operator, which uses a badly written
+CUDA kernel to calculate the histogram. The purpose here is not to show-case
+how the write CUDA kernels, but to present how to use CUDA kernels within
+DALI operators.
```

Comment:
I am not sure about "badly" written. For me, naive is descriptive enough. It clearly states that this is just an example of some CUDA code. Or maybe something like "not optimized" might work?

Reply:
Done